documentation/api/io.strimzi.api.kafka.model.KafkaAuthorizationOpa.adoc

```diff
@@ -0,0 +1,44 @@
+To use link:https://www.openpolicyagent.org/[Open Policy Agent^] authorization, set the `type` property in the `authorization` section to the value `opa`.
+The Open Policy Agent authorizer has several configuration options:
+
+`url`::
+The URL used to connect to the Open Policy Agent server.
+The URL has to include the policy which will be queried by the authorizer.
+**Required.**
+
+`allowOnError`::
+Defines whether Kafka client should be allowed or denied by default when the authorizer fails to query the Open Policy Agent (for example because it is temporarily unavailable).
+Defaults to `false` - all actions will be denied.
+
+`initialCacheCapacity`::
+Initial capacity of the local cache used by the authorizer.
+Defaults to `5000`.
+
+`maximumCacheSize`::
+Maximum capacity of the local cache used by the authorizer.
+Defaults to `50000`.
+
+`expireAfterMs`::
+The expiration of the records kept in the local cache.
+In milliseconds.
+Defaults to `3600000` milliseconds (1 hour).
```

Comment:
maybe this above definitions could be used to enrich the descriptions in the model as I raised before.